src/dotless.Core/Parser/Tree/Ruleset.cs

```diff
@@ -279,6 +281,50 @@ protected void EvaluateRules(Env env)
             env.Frames.Pop();
         }
 
+        /// <summary>
+        /// Merges properties with merge operators '+' and '+_'
+        /// </summary>
+        /// <returns>New merged ruleset</returns>
+        private NodeList MergeProperties()
+        {
+            var groups = new Dictionary<string, List<Rule>>();
+            var sorted = new List<Node>();
+            foreach (var node in Rules)
+            {
+                var rule = node as Rule;
+                if (rule == null || rule.Merge == "")
```

Comment:
I will look at it, but if a later rule overwrites the merged property ordering is important so I did not wish to break it apart.

For a static stylesheet you could get away with it but if the stylesheet is in multiple parts that are joined before parsed you might have overlapping rules both setting the same property and I elected to play it safe in this case.

But I will experiment with some more unit tests to see if I can design one that exemplifies this and then experiment with it and see if I can solve it better.